README.md

```diff
@@ -16,20 +16,20 @@ Once installed, clone the repository and install its dependencies running:
 
     $ npm install
 
-## Running the examples
-In order to run the different examples, open the folder with the name of the flow you want to try out, and run its `app.js` file. For instance, to run the Authorization Code example do:
-
-    $ cd authorization_code
-    $ node app.js
-
-Then, open `http://localhost:8888` in a browser.
-
 ### Using your own credentials
-The examples contains a working client ID and secret key. Note, however, that they might be rate limited if they are used frequently. If you are planning to create an application, we recommend you register your app and get your own credentials instead of using the ones in this project.
+You will need to register your app and get your own credentials from the spotify developer dashboard.
 
-Go to [My Applications on Spotify Developer](https://developer.spotify.com/my-applications) and create your application. For the examples, we registered these Redirect URIs:
+To do so, go to [My Applications on Spotify Developer](https://developer.spotify.com/my-applications) and create your application. For the examples, we registered these Redirect URIs:
```

Comment:
Change to `[your Spotify for Developers Dashboard](https://beta.developer.spotify.com/dashboard)`